nuxeo-core/nuxeo-core-io/src/main/java/org/nuxeo/ecm/core/io/marshallers/json/AbstractJsonReader.java

```diff
@@ -144,4 +146,59 @@ protected String getStringField(JsonNode jn, String elName) {
         }
     }
 
+    /**
+     * Try to get a boolean property of the given {@link JsonNode}. Return null if the node is null or not a boolean.
+     * @param jn The {@link JsonNode} to parse.
+     * @param elName The property name.
+     * @return The boolean value if it exists and is a boolean property, null otherwise.
```

Comment:
Note that the verbs should be descriptive and not imperative: `Tries to get`..., `Returns null`...
`null` is a literal so should be denoted as `{@code null}`.
For the `param`/`return` don't need an initial capital or final period: `the property name`.

See style guide at http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#styleguide